app/views/locations/show.html.erb

```diff
@@ -45,3 +45,23 @@
 <p>
   You can also book in person by visiting any Citizens Advice bureau.
 </p>
+
+<% content_for :sidebar do %>
```

Comment:
Because it is only required when viewing a location, not for all other location pages.